src/component/page/home/index.tsx

```diff
@@ -4,13 +4,14 @@ import { useContext, useState } from "react";
 import {
   DefaultHomeProps,
   PlasmicHome,
+  PlasmicHome__VariantMembers,
 } from "component/plasmic/shared/PlasmicHome";
 import { UserContext } from "context/UserContext";
 import { IUpdate } from "module/interface/update";
 
 type VariantType = "isEmpty" | "isTimeline" | "isVenture" | undefined;
 type IsActive = "feed" | "settings" | "members" | "isNewVenture" | undefined;
-type IsVisible = "postDetails" | "mobileSidebar" | "showModal" | undefined;
+export type IsVisible = PlasmicHome__VariantMembers["isVisible"] | undefined;
```

Comment:
Exporting this here so we can use it in other components while keeping in sync with Plasmic.